tests/unit/env_variables/test_environment_variables.py

```
@@ -0,0 +1,72 @@
+import os
+import pytest
+import requests
+
+
+class TestEnvironmentVariables:
+
+    """Tests for expiring environment variables."""
+
+    @pytest.mark.xfail(
+        condition=lambda: os.getenv("FLASK_DEBUG") == "1",
+        reason="Skipping in debug mode",
+        run=False,
+    )
+    def test_hugging_face_token(self):
+        """Test if HUGGING_FACE_TOKEN is valid by querying Hugging Face API."""
+        token = os.getenv("HUGGING_FACE_TOKEN")
+        if not token:
+            pytest.skip("Skipping test: HUGGING_FACE_TOKEN is not set")
```

__question, blocking:__ Is this the correct approach here?

Based on the Pytest mark you added, I'd expect this line to be rendered obsolete when the API is run in debug mode. Otherwise, I expect we're running in the prod environment, and in that case, this test should fail, perhaps with a message saying that the token was not found.

---

Modified code with assert instead of pytest.skip